.github/workflows/upload-docs-ios.yml

```diff
@@ -0,0 +1,51 @@
+name: Upload Docs iOS
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - 'test-doc-*'
+
+jobs:
+  upload-docs:
+
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout current branch
+      uses: actions/checkout@v4
+    - name: Install Jazzy
+      run: |
+        gem install jazzy
+    - name: Generate docs
+      run: |
+        jazzy
+        cp -r docs docs_new
+    - name: Checkout docs branch
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: docs
+        clean: false
+    - name: Commit documentation changes
+      run: |
+          if [ "$(git status --porcelain | wc -l)" -gt 0 ]; then
+            git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+            git config --local user.name "github-actions[bot]"
+            git status
+            TAG_NAME=${{ github.ref }}
+            TAG_NAME=${TAG_NAME#refs/tags/}
+            echo "Current tag: $TAG_NAME"
+            rm -rf docs
```

Comment:
Same comment as on the android side... would like to see some more defensive coding here -- check the size of `docs_new` before removing docs. Thanks.